administrator/components/com_content/views/article/tmpl/edit.php

```diff
@@ -181,11 +181,15 @@
 	</div>
 </form>
 <?php
+JHtml::_('jquery.framework', false);
 $document = JFactory::getDocument();
 $document->addScriptDeclaration('
-function copytext() {
-textBox = document.getElementById("articletitle");
-message = document.getElementById("menutitle");
-message.value = textBox.value;
+$(document).ready(function () {
+$("input[name=title]").keyup(function () {
```

Comment:
Instead of this use `http://api.jquery.com/on/` instead of `keyup`.

    $("a").on() --> Provides all functionality required for attaching event handlers. (Newest in jQuery 1.7)